### PR TITLE
TP2000-156 - Component sequence number start at 1, fixing the importer.

### PR DESCRIPTION
**Pull request by stuaxo**
Component sequence numbers start at 1 not 0, fixing the importer.
Make `test_measure_form_wizard_create_measures` a little more batch oriented making it easier to test this, should save some queries too.

### CODE REVIEW

File: measures/tests/test_views.py [hunk 1]
```diff
@@ -413,64 +414,60 @@ def test_measure_form_wizard_create_measures(
...
+        measures.values_list(
+            "pk", "conditions__component_sequence_number", "conditions__condition_code"
         )
-    )
+    ) == [
+        (measure_data[0].pk, 1, condition_code1.pk),
+        (measure_data[0].pk, 2, condition_code2.pk),
+        (measure_data[1].pk, 1, condition_code1.pk),
+        (measure_data[1].pk, 2, condition_code2.pk),
+    ]
```
Comment (simonwo) [inline measures/tests/test_views.py:457]: Using lists here is asserting an ordering, whereas we are not asking for any specific ordering in the query, which looks brittle. Would a set be a better choice?

File: measures/tests/conftest.py [hunk 2]
```diff
@@ -361,6 +361,11 @@ def measure_type():
     )
 
 
+@pytest.fixture()
+def erga_omnes():
+    return factories.GeoGroupFactory(area_id=1011)
+
+
```
Comment (simonwo) [inline measures/tests/conftest.py:368]: Where is this used?

Comment (stuaxo) [reply measures/tests/conftest.py:368]: Good catch I believe this came from a previous version of this where we were building a different test for this work, will remove, I'm can imagine this fixture coming back in the future though :)

Comment (stuaxo) [reply measures/tests/conftest.py:368]: Removed.

**Review by simonwo — APPROVED** [review]

**Review by gabelton — APPROVED** [review]

**Pull request merged**